### PR TITLE
Lite: Unpack quantization support added

### PR DESCRIPTION
**Pull request by ANSHUMAN87**
Quantization support added for Unpack Op

### CODE REVIEW

**Comment by ANSHUMAN87** [comment]
Add ref: #27760

**Comment by jdduke** [comment]
We should also bump the operator version when using quantized types. See also [this commit](https://github.com/tensorflow/tensorflow/commit/e476315d4484ed2b344983e1c5258b976927257c) for an example of how to bump version(s) when adding new features/types to an existing operator.

**Comment by ANSHUMAN87** [comment]
> We should also bump the operator version when using quantized types. See also [this commit](https://github.com/tensorflow/tensorflow/commit/e476315d4484ed2b344983e1c5258b976927257c) for an example of how to bump version(s) when adding new features/types to an existing operator.

@jdduke : Your comment is addressed now, please check, Thanks!

**Comment by jdduke** [comment]
Adding @suharshs and @jianlijianli for review.

File: tensorflow/lite/kernels/unpack.cc [hunk 1]
```diff
@@ -79,6 +80,11 @@ TfLiteStatus Prepare(TfLiteContext* context, TfLiteNode* node) {
     TfLiteIntArray* copied_output_shape = TfLiteIntArrayCopy(output_shape);
     TfLiteTensor* output = GetOutput(context, node, i);
     TF_LITE_ENSURE_EQ(context, output->type, input->type);
+    // Guarantee input/output quantization params match as we do not support
+    // unpacking quantized tensors.
```
Comment (jdduke) [inline tensorflow/lite/kernels/unpack.cc]: This comment is slightly confusing, checking quantization match makes sense, but "we do not support unacking quantized tensors" doesn't sound right, as that's exactly what we're doing? Did you mean rescaling quantized tensors?

**Review by jdduke — CHANGES_REQUESTED** [review]

**Comment by jdduke** [comment]
Looks good, thanks!

**Comment by ANSHUMAN87** [comment]
@rthadur : Would you please help proceed with this PR, Thanks!

**Comment by gbaned** [comment]
@suharshs, @jianlijianli Could you PTAL and approve.

**Review by jianlijianli — DISMISSED** [review]
Thanks for adding this. Logic looks great. Please address the comment jdduke has. Thanks!

**Comment by ANSHUMAN87** [comment]
@jdduke : Your comment is addressed now, Thanks!
@jianlijianli : Thanks for approving the PR!

**Review by jianlijianli — DISMISSED** [review]
Thanks for the update.

**Comment by ANSHUMAN87** [comment]
@gbaned : I have resolved the merge conflict, please help proceed with this PR, thanks!

**Comment by ANSHUMAN87** [comment]
@gbaned : Would you please help proceed with this PR, thanks!

**Review by jdduke — APPROVED** [review]

**Pull request merged**